crates/pixi_global/src/common.rs

```diff
@@ -87,7 +87,6 @@ impl BinDir {
     /// `bin_dir` with the provided `exposed_name`. If the target platform is
     /// Windows, it sets the file extension to `.exe`.
     pub(crate) fn executable_trampoline_path(&self, exposed_name: &ExposedName) -> PathBuf {
-        // Add .bat to the windows executable
         let exposed_name = if cfg!(windows) {
```

Comment:
This doc change is on purpose. It's not valid anymore, and the comment wasn't all that useful